python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_network_services.schema.yml

```diff
@@ -826,6 +826,200 @@ $defs:
+                    mode:
+                      type: str
+                      description: |-
+                        Port-Channel mode.
+                        Should not be set on Port-Channel subinterfaces.
+                      valid_values:
+                        - "active"
+                        - "passive"
+                        - "on"
+                      default: "active"
+                    member_interfaces:
+                      description: |-
+                        Port-Channel member interfaces.
+                        Should not be set on Port-Channel subinterfaces.
+                      type: list
+                      primary_key: name
+                      items:
+                        type: dict
+                        keys:
+                          name:
+                            type: str
+                            required: true
+                            description: |-
+                              Ethernet interface name like 'Ethernet2'.
+                              Member interface cannot be subinterface.
+                            pattern: "Ethernet[\\d/]+"
+                          description:
+                            type: str
+                            description: |-
+                              Interface description for this member.
+                              If not set, a default description will be configured with '[<peer>[ <peer_interface>]]'.
+                          peer:
+                            type: str
+                            description: |-
+                              The peer device name. Used for description and documentation.
+                              If not set, this inherits the peer setting on the port-channel interface.
+                          peer_interface:
+                            type: str
+                            description: |-
+                              The peer device interface. Used for description and documentation.
+                          speed:
+                            type: str
+                            description: |-
+                              Speed should be set in the format `<interface_speed>` or `forced <interface_speed>` or `auto <interface_speed>`.
+                          structured_config:
+                            type: dict
+                            documentation_options:
+                              hide_keys: true
+                            description: |-
+                              Custom structured config for the member ethernet interface.
+                            $ref: "eos_cli_config_gen#/keys/ethernet_interfaces/items"
+                    ip_address:
+                      type: str
+                      description: IPv4 address/Mask.
+                    ip_addresses:
+                      type: list
+                      items:
+                        type: str
+                        description: IPv4_address/Mask.
```

Comment:
i think we need to use one or the other, but not both. Since the rest of the model allows only a single value - like peer and peer-interface, I would prefer to do the same here. It quickly becomes cluttered if we wish to map out member interfaces etc for each node. Then it is better to just add more entries.